src/lib/select/select.scss

```diff
@@ -6,7 +6,6 @@
 
 $mat-select-arrow-size: 5px !default;
 $mat-select-arrow-margin: 4px !default;
-$mat-select-panel-max-height: 256px !default;
```

Comment:
We should leave the variable in there, because somebody could be depending on it.